modules/kueezRtbBidAdapter.js

```diff
@@ -0,0 +1,276 @@
+import { _each, deepAccess, parseSizesInput, parseUrl, uniques, isFn } from '../src/utils.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+const DEFAULT_SUB_DOMAIN = 'exchange';
+const BIDDER_CODE = 'kueezrtb';
+const BIDDER_VERSION = '1.0.0';
+const CURRENCY = 'USD';
+const TTL_SECONDS = 60 * 5;
+const UNIQUE_DEAL_ID_EXPIRY = 1000 * 60 * 15;
+export const SUPPORTED_ID_SYSTEMS = {
+  'britepoolid': 1,
+  'criteoId': 1,
+  'id5id': 1,
+  'idl_env': 1,
+  'lipb': 1,
+  'netId': 1,
+  'parrableId': 1,
+  'pubcid': 1,
+  'tdid': 1,
+  'pubProvidedId': 1
+};
+const storage = getStorageManager({ gvlid: null, bidderCode: BIDDER_CODE });
```

Comment:
Why is gvlid specifically null here instead of just not being included? I have seen this in a few id modules but usually bid adapters either add the gvlid or omit the parameter altogether.

Reply:
@ChrisHuie When we submitted this PR we didn't had a gvlid.
It's there just we wont forget to put it there once we have it 😀